packages/katello/katello/katello.spec

```diff
@@ -36,6 +36,10 @@ Requires: %{?scl_prefix}rubygem-hammer_cli
 Requires: %{?scl_prefix}rubygem-hammer_cli_foreman
 Requires: %{?scl_prefix}rubygem-hammer_cli_katello
 
+# katello-agent requirements
+Requires: qpid-cpp-server-linearstore
+Requires: qpid-cpp-client-devel
```

Comment:
is client-devel actaully a run time requirement?

Reply:
Thats a good question

Reply:
oh i see this is actually down below, is it actually required?

Reply:
One way to find out, removed.